client/components/additionalFeatures.jsx

```diff
@@ -37,6 +37,7 @@ const H2 = styled.h2`
   font-size: 40px;
   line-height: 48px;
   letter-spacing: -1px;
+  textAlign: center;
```

Comment:
I think this style might not work. I believe it should be text-align here. As I understand, camel case for CSS properties is for React only.